src/tools/roc_recv/cmdline.ggo

```diff
@@ -42,8 +42,8 @@ section "Options"
     option "packet-limit" - "Maximum packet size, in bytes"
         int optional
 
-    option "frame-size" - "Internal frame size, number of samples"
-        int optional
+    option "frame-size" - "Duration of the internal frame size, in nanoseconds"
+        typestr="TIME" string optional
```

Comment:
`Duration of the internal frame size` -> `Duration of the internal frames`

Reply:
done.